openapi.go

```diff
@@ -16,10 +16,15 @@ import (
 
 func NewOpenAPI() *OpenAPI {
 	desc := NewOpenApiSpec()
+
+	mp := NewMetadataParsers()
+	mp.Initialize(DefaultParsers)
```

Comment:
Since you use `NewMetadataParsers` AND `Initialize` function, why don't you write something like `mp := NewMetadataParsers(DefaultParsers)` directly? Since it makes sense in a business point of view, I think it's a correct API.